src/sgame/sg_admin.cpp

```diff
@@ -3897,6 +3893,15 @@ bool G_admin_listplayers( gentity_t *ent )
 		           ( !authed ) ? "^1NOT AUTHED" : "" ) );
 	}
 
+	ADMBP( va( "\n^3listplayers:^* legend:" ) );
+
+	if ( canset ){
```

Comment:
nit style:
```
if ( canset )
{
    ...
}
```

Reply:
fixed in https://github.com/Unvanquished/Unvanquished/pull/2349/commits/56ffb617c5896ffec35ba73491c22465259e4d14